vlog4j-core/src/main/java/org/semanticweb/vlog4j/core/reasoner/implementation/VLogReasoner.java

```diff
@@ -808,4 +834,4 @@ ReasonerState getReasonerState() {
 	void setReasonerState(ReasonerState reasonerState) {
 		this.reasonerState = reasonerState;
 	}
-}
+}
```

Comment:
please do not remove the newline character at the end of a .java file, it is required (for parsing, I think)

Reply:
For what it's worth, it's “required” because POSIX mandates that every line needs a terminating LF, and some really old tools may not work correctly on files not having those (as far as I know, we don't use any of them; line counts using `wc` might be off by one, but that's about it).

As long as we're still happily mixing UNIX-style LF terminators with Windows-style CRLF terminators in the same codebase, I wouldn't put too much emphasis on sticking to such antiquated requirements.

Reply:
Ok, I see.